docker/local-metadata/docker-compose-dev-postgres.yml

```diff
@@ -11,18 +11,11 @@
 
 version: "3.9"
 services:
-  postgres:
-    build:
-      context: ../../.
-      dockerfile: docker/local-metadata/Dockerfile_postgres
-    container_name: openmetadata_postgres
```

Comment:
Why are we not using our image? In there we should be having the latest postgres already but with the users ready

Reply:
I have aligned this file with what we have in `docker-compose-dev.yml`. I can roll back this change and align it with `docker-compose-dev.yml`. What should I do?

Reply:
LGTM then, thanks for raising

Reply:
We have agreed offline to leave this change.